presto-hive/src/main/java/com/facebook/presto/hive/parquet/ParquetTypeUtils.java

```diff
@@ -246,6 +246,25 @@ public static ParquetEncoding getParquetEncoding(Encoding encoding)
         return null;
     }
 
+    public static ColumnIO getColumnIOByName(GroupColumnIO groupColumnIO, String name)
```

Comment:
Let's add a javadoc comment here (and remove the comments below in the method body):

```
/**
 * Parquet column names are case-sensitive unlike Hive, which converts all column names to lowercase.
 * Therefore, when we look up columns we first check for exact match, and if that fails we look for a case-insensitive match.
 */
```